arches/app/media/js/views/components/plugins/etl-manager.js

```diff
@@ -1,33 +1,151 @@
 define([
+    'jquery',
     'knockout',
     'arches',
-], function(ko, arches) {
+], function($, ko, arches) {
     return ko.components.register('etl-manager', {
         viewModel: function(params) {
-            var self = this;
+            const self = this;
             this.loading = params.loading;
             this.alert = params.alert;
             this.loading(true);
             this.selectedModule = ko.observable();
+            this.activeTab = ko.observable();
+            this.isImport = ko.observable(true);
+            this.loadEvents = ko.observable();
+            this.selectedLoadEvent = ko.observable();
+            this.validated = ko.observable();
+            this.validationError = ko.observableArray();
+
+            this.selectedLoadEvent.subscribe(function(val){
+                self.fetchValidation(val.loadid);
+            });
+            this.moduleSearchString = ko.observable('');
+            this.taskSearchString = ko.observable('');
+            this.tabs = [
```

Comment:
Not really necessary to change this now, but the titles should eventually come from the backend so that they're easier to translate.